### PR TITLE
Refactoring swagger artifact generation

### PR DESCRIPTION
**Pull request by poojamaiya**
#### Summary
Refactors swagger artifact generation to check in the client along with the software. This way, we don't have to explicitly generate the client for the tests. Also changed the directory holding the generated artifacts from ./handler/v1/api to ./swagger/v1/generated - Addressing issue #115.

#### Implementation details
Check in client (in addition to the models) with the software and change tests to use this client.

#### Testing

- [X] cluster-state-service binary built locally and unit-tests pass (`cd cluster-state-service; make; cd ../`)
- [X] cluster-state-service build in Docker succeeds (`cd cluster-state-service; make release; cd ../`)
- [X] daemon-scheduler binary built locally and unit-tests pass (`cd daemon-scheduler; make; cd ../`)
- [X] daemon-scheduler build in Docker succeeds (`cd daemon-scheduler; make release; cd ../`)

New tests cover the changes: Yes

#### Description for the changelog
Refactoring swagger artifact generation to check in the client along with the software

#### Licensing

This contribution is under the terms of the Apache 2.0 License: Yes

#### Before merging

- [X] cluster-state-service end-to-end tests pass. Required setup details in cluster-state-service/internal/Readme.md
- [X] daemon-scheduler integration tests pass.
- [X] daemon-scheduler end-to-end tests pass.

### CODE REVIEW

File: cluster-state-service/Makefile [hunk 1]
```diff
@@ -19,10 +19,10 @@ ROOT := $(shell pwd)
 .PHONY: all
 all: clean generate build unit-tests
 
-.PHONY: generate-models
-generate-models:
-	./scripts/v1/generate_swagger_models.sh
-	@echo "Generated swagger models"
+.PHONY: generate-swagger-artifacts
+generate-swagger-artifacts:
```
Comment (tuedtran) [inline cluster-state-service/Makefile]: Should we make it to regenerate swagger artifacts when swagger.json has changed like we did for the DS? https://github.com/blox/blox/blob/dev/daemon-scheduler/Makefile#L37

**Comment by kylbarnes** [comment]
Can you make the following changes?

* Sync up `generate_swagger_artifacts.sh` between `daemon-scheduler` and `cluster-state-service`. Is there any reason these can't be the same files, or close to it? Also, please choose a common directory (`scripts/` vs `scripts/v1/`).
* Move the daemon-scheduler swagger artifacts into `daemon-scheduler/swagger/v1/` for consistency.
* Do we need `daemon-scheduler/pkg/clients/css/` anymore? Can we just reference these directly at `cluster-state-service/swagger/v1/generated/`?

File: cluster-state-service/Makefile (hunk 1 above)
Comment (poojamaiya) [inline cluster-state-service/Makefile]: Thanks, done.

**Comment by poojamaiya** [comment]
@kylbarnes 
* Sync up generate_swagger_artifacts.sh between daemon-scheduler and cluster-state-service. Is there any reason these can't be the same files, or close to it? Also, please choose a common directory (scripts/ vs scripts/v1/). - **_DONE_** _(Changed to scripts/v1)_
* Move the daemon-scheduler swagger artifacts into daemon-scheduler/swagger/v1/ for consistency. - **_DONE_**
* Do we need daemon-scheduler/pkg/clients/css/ anymore? Can we just reference these directly at cluster-state-service/swagger/v1/generated/? - **_DONE_** (_Changed DS to use the CSS client that is checked in with CSS)_

**Comment by kylbarnes** [comment]
Can you also update the swagger.json relative urls in `cluster-state-service/README.md` and `daemon-scheduler/README.md`?

**Comment by poojamaiya** [comment]
@kylbarnes  - Good catch! Updated swagger.json relative urls in cluster-state-service/README.md and daemon-scheduler/README.md. Thanks!

**Pull request merged**